bigtop_toolchain/bin/puppetize.sh

```diff
@@ -81,7 +81,7 @@ case ${ID}-${VERSION_ID} in
         dnf config-manager --set-enabled codeready-builder-for-rhel-8-rhui-rpms
         ;;
     openEuler-*)
-        dnf -y install hostname curl sudo unzip wget ruby ruby-devel vim systemd-devel findutils 'dnf-command(config-manager)' nc initscripts openeuler-lsb openssl-devel make gcc-c++ openEuler-rpm-config python3-pip python3-devel dbus
+        dnf -y install hostname curl sudo unzip wget ruby ruby-devel vim systemd-devel findutils 'dnf-command(config-manager)' nc initscripts openeuler-lsb openssl-devel make gcc-c++ openEuler-rpm-config python3-pip python3-devel dbus python3-unversioned-command
```

Comment:
puppetize.sh should not have resources unrelated to Puppet setup . It the package is required for building pruducts, Puppet manifest of bigtop_toolchain should be updated. If it is only required for smoke-tests, `deploy/puppet/manifests/python.pp` should be updated. @MacChen01

Reply:
ok, the code has modifed:
https://github.com/apache/bigtop/pull/1244/commits/69d371579597f35f19691b552faf7d1c678ebdb8